privacy-contracts/src/main/solidity/OnChainPrivacyGroupManagementProxy.sol

```diff
@@ -1,5 +1,6 @@
 pragma solidity ^0.5.12;
 
+pragma solidity ^0.5.9;
```

Comment:
Which version is the required one? We shouldn't have two :P

Reply:
My local version does only have the 0.5.9 .... Weird!